samples/nrf9160/modem_shell/src/main.c

```diff
@@ -76,6 +76,13 @@ K_MUTEX_DEFINE(at_resp_buf_mutex);
 
 K_SEM_DEFINE(nrf_carrier_lib_initialized, 0, 1);
 
+void nrf_modem_fault_handler(struct nrf_modem_fault_info *fault_info)
+{
+	printk("Modem error: 0x%x, PC: 0x%x\n", fault_info->reason, fault_info->program_counter);
```

Comment:
`Modem error:` somehow indicates it could be some other error than crash too.
So, I'd put `Modem crash error code:` or `Modem crash code:` or `Modem crash reason:`

Reply:
I have to say that I tend to agree. That line is actually copied from nRF Modem library, but we can modify it to make it more clear. I like `Modem crash reason:` the most.

Reply:
Changed the text.